### ISSUE TITLE
DefaultFirebaseOptions is missing some properties.

### ISSUE THREAD

**Issue by FranRiadigos** [invertase/flutterfire_cli#21]
After updating with the new installation flow some resources are not retrieved properly.

The following is an error thrown from the `Firebase Auth` **emulator** when signing in with Google.

`The Auth Emulator only support sign-in with google.com using id_token, not access_token. `

The `id_token` retrieved is null.

It turns out the Firebase configuration is missing some OAuth client ids, besides, I realized the `ios` configuration in the `DefaultFirebaseOptions` contains `androidClientId`, whereas the `android` one doesn't.

Switching back to use the default `google-services.json` manual installation works, so it's not a blocker.

I mainly got this error from the **emulator**, didn't try with a real instance.

**Comment by Salakar** [comment]
Which properties are missing? androidClientId is correct in ios, see reference documentation - it's a ios only field use for cross-platform deep linking in dynamic links.

**Comment by FranRiadigos** [comment]
I'm sorry, didn't see this reply in my inbox.

Tbh, something is missing but not sure what it is, otherwise `Firebase Auth` wouldn't complain and the `id_token` would not be null.

I'll make some experiments and will let you know more.

**Comment by pr-Mais** [comment]
Are you using [google_sign_in](https://pub.dev/packages/google_sign_in) plugin to retrieve the Google auth credentials?

**Comment by FranRiadigos** [comment]
@pr-Mais Yes, I am, and I saw on the documentation this is not supported.

**Comment by pr-Mais** [comment]
Yes, so you might still need manual platform configurations until it's supported, related issue https://github.com/flutter/flutter/issues/96391

**Comment by Salakar** [comment]
Closing as this is not related to CLI usage but rather google sign-in, as above
